chapters/scoping.tex

```diff
@@ -759,13 +747,13 @@ \subsection{Generation of the flat equation system}\label{generation-of-the-flat
   conditional declaration expressions evaluate to true or it is a
   constant in a package.
   \begin{nonnormative}
-  Conditional components can be used in connect-statements, and if the component is conditionally disabled the connect-statement is removed.
+  Conditional components can be used in connect-equations, and if the component is conditionally disabled the connect-statement is removed.
```

Comment:
I think connect should be treated similarly as transition here.

Reply:
Like above, I basically agree, but would prefer to do it globally in a separate PR.  Or do you see a reason to not be consistent?

Reply:
Filed #2713 to settle this.  Enough to resolve?